cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/GoalViolationDetector.java

```diff
@@ -51,7 +51,9 @@
   private static final Logger LOG = LoggerFactory.getLogger(GoalViolationDetector.class);
   private final List<Goal> _detectionGoals;
   private ModelGeneration _lastCheckedModelGeneration;
-  private final Pattern _excludedTopics;
+  private final Pattern _excludedTopicsPattern;
+  private final Pattern _mustHaveLeaderReplicaPerBrokerTopicsPattern;
+  private final int _minLeaderReplicaCountForTopicsMustHaveLeaderReplicaPerBroker;
```

Comment:
`_minLeaderReplicaCountForTopicsMustHaveLeaderReplicaPerBroker` is assigned but never accessed.

Does your IDE not warn you about this?

Reply:
Oops, missed it